test/check-application

```diff
@@ -34,7 +34,6 @@ class TestApplication(testlib.MachineCase):
         # run a container (will exit immediately)
         m.execute("podman run -d --name swamped-crate busybox sh")
 
-        # FIXME: UI needs to listen to change signals and update automatically
```

Comment:
This is still true, please keep it. It is really wrong to have to reload the page after state changes, but until podman has a way to notify clients about changes, it's a necessary workaround.